HW04/aleksei_zhuchkov/spec/notification_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../lib/notification'
+
+RSpec.describe Notification do
+  let(:notification) { described_class.new('test title', 'test description') }
+
+  it 'return title' do
+    expect(notification.title).to eq('test title')
+  end
+
+  it 'return description' do
+    expect(notification.description).to eq('test description')
+  end
+
+  describe 'method notification' do
+    it 'when notification doesn\'t add' do
+      expect(notification.get_notifications([])).to eq([])
+    end
+
+    it 'when add some notification' do
+      test_arr = []
```

Comment:
do not create variables this way

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/42b91004881bb9ff8ce15245c569d4c34f0512f5